set.mm

```diff
@@ -281015,6 +281057,27 @@ the same dimension over the same (nonzero) ring.  (Contributed by AV,
       DUEHIBCDEFGJKLMTUAABBCHINOUBUC $.
   $}
 
+  ${
+    $d f I $.  $d x y ph $.  $d x y R $.  $d x y X $.  $d y Y $.
+    psraddclg.s $e |- S = ( I mPwSer R ) $.
+    psraddclg.b $e |- B = ( Base ` S ) $.
+    psraddclg.p $e |- .+ = ( +g ` S ) $.
+    psraddclg.r $e |- ( ph -> R e. Mgm ) $.
+    psraddclg.x $e |- ( ph -> X e. B ) $.
+    psraddclg.y $e |- ( ph -> Y e. B ) $.
+    $( Closure of the power series addition operation.  (Contributed by SN,
+       12-Apr-2025.) $)
+    psraddclg $p |- ( ph -> ( X .+ Y ) e. B ) $=
```

Comment:
Better call this ~psraddcld (because this theorem is in deduction form)

Reply:
Given none of the many adjacent theorems (like psrmulcl) have the d suffix this seems to be the 'theorems far into main set.mm with many hypotheses may elide the suffix' thing. I don't remember the exact wording but it also applies to theorems like dvmul. Either way I would like to defer this change to another pr

Reply:
> Given none of the many adjacent theorems (like psrmulcl) have the d suffix this seems to be the 'theorems far into main set.mm with many hypotheses may elide the suffix' thing. I don't remember the exact wording but it also applies to theorems like dvmul. Either way I would like to defer this change to another pr

The same argumentation would also hold for suffix "g"...

As I wrote before, I would prefer to replace/revise ~psraddcl accordingly, instead of adding a new theorem ~psraddclg. Maybe a new theorem

`grpmgm $p |- ( G e. Grp -> G e. Mgm ) $=`

would be helpful to combine the currently used two steps grpmnd and mndmgm, and maybe addional proofs could be shortened.

Reply:
>Maybe a new theorem
> 
> `grpmgm $p |- ( G e. Grp -> G e. Mgm ) $=`
> 
> would be helpful to combine the currently used two steps grpmnd and mndmgm, and maybe addional proofs could be shortened.

The new ~grpmgmd is even better.